vcr/api/v2/api.go

```diff
@@ -229,3 +231,34 @@ func (w *Wrapper) CreateVP(ctx echo.Context) error {
 	}
 	return ctx.JSON(http.StatusOK, vp)
 }
+
+// VerifyVP handles API request to verify a Verifiable Presentation.
+func (w *Wrapper) VerifyVP(ctx echo.Context) error {
```

Comment:
The number of checks do not represent the number of 400 errors that are returned by this code. It seems some are returned as 500s?

Reply:
This could simply be fixed by mapping the validationError to a status code.

Reply:
400 is returned when `Bind()` fails (see `github.com/labstack/echo/v4@v4.6.3/bind.go:73`)